index.js

```diff
@@ -41,40 +43,52 @@ function checkGuess() {
 
   if (guess !== targetNumber) {
     if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
+      if(guess <= 0){
+        belowZeroMessage.style.display = 'block';
+      }else {
+      tooLowMessage.style.display = 'block';
+    }
     } else {
-      tooLowMessage.style.display = '';
+      if(guess>=100){
+        aboveHundredMessage.style.display = 'block';
+      }else{
+      tooHighMessage.style.display = 'block';
     }
+    } 
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
 
-    numberOfGuessesMessage.style.display = '';
+    numberOfGuessesMessage.style.display = 'block';
+    if(attempts===4){
+      numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guess remaining`;
+    } else{
     numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
-  }
+  }}
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if (attempts === maxNumberOfAttempts) {
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
   guessInput.value = '';
 
-  resetButton.style.display = '';
+  resetButton.style.display = 'block';
 }
 
 function hideAllMessages() {
-  for (let elementIndex = 0; elementIndex <= messages.length; elementIndex++) {
+  for (let elementIndex = 0; elementIndex < messages.length; elementIndex++) {
     messages[elementIndex].style.display = 'none';
   }
 }
 
-funtion setup() {
+function setup() {
```

Comment:
Fix: Missing requirement:
>  - [ ] When you click the "reset" button, the form should return to its initial state (not disabled)

After I click "reset", the submit button stays disabled (it stays disabled in both scenarios: if I correctly guessed the number OR if I hit the max number of attempts)!
![](https://github.com/butolinka/intro-to-programming-section-5/assets/22607722/799f7792-46fd-4449-a91a-503148323ad8)

![](https://github.com/butolinka/intro-to-programming-section-5/assets/22607722/ee311bc4-2249-44c9-99a7-5583b7db781e)